ng/src/web/pages/portlists/dialog.js

```diff
@@ -76,8 +72,11 @@ const PortListsDialog = ({
   );
 
   const data = {
-    ...DEFAULTS,
     ...port_list,
```

Comment:
Would be better to pass the necessary values from  port_list explicitly. But I guess we only need the id which also should be put in the defaultValues object instead into the values object.